inference-engine/ie_bridges/python/requirements.txt

```diff
@@ -1,2 +1,2 @@
-numpy
-cython>=0.29
+numpy==1.18.1
```

Comment:
We need to align numpy version with the version in MO requirements if it specified there

Reply:
As I see in MO requirements, there is `numpy>=1.13.0`  version required.

Reply:
Well, then everything is OK :)